### ISSUE TITLE
Update Home Assistant Integration for HERU Firmware v1.14

### ISSUE THREAD

**Issue by azachar** [toringer/home-assistant-heru#48]
The current Home Assistant integration for Östberg HERU units may not fully support the latest features and modifications introduced in firmware version 1.14. This could lead to limited functionality or compatibility issues for users who have updated their HERU systems to the newest firmware.

For more details on the latest firmware release, please refer to the HERU Firmware Release Notes.

https://fsp.ostberg.com/products/front/products/download_document/36321/4387/firmware-heru-iqc

Best regards,
Andrej

PS: Thank you for your great work, if needed happy to help with the update.

**Comment by magicus** [comment]
Can you summarize what changes are present in the new update that conflicts with the current integration? The changelog was quite long and verbose, and from a glance it was hard to see what could cause failures.

**Comment by magicus** [comment]
I've read through the changelog document now. It seems that all possible modbus register changes happened in 1.10; from there on there are no described changes that could cause issues with modbus.

On the other hand, the major breaking change of introducing decimals in the temperature values was introduced in 1.09j, and is not even mentioned in the changelog! So maybe it is not possible to trust the documentation to any large extent.

It would be interesting to know what firmware version other users are using, and if they are safe.

**Comment by magicus** [comment]
I created a support thread at https://community.home-assistant.io/t/ostberg-heru-energy-recovery-unit-ftx-component/810761/2. Hopefully we can get some input from folks what versions are tested.

**Comment by azachar** [comment]
Dear Magnus,

Thank you for your quick review!

What is missing so far? Seems like bunch of new statuses that can be exposed, and are not exposed yet, such as 1x00031 (effective recovery status).


Only breaking change I noticed in Heru's changelog is 
1x00039->1x00037 (but you were not using it anyway).

So far it seems working fine, I m just running it with v1.14, before I was using v1.10 and it was working pretty good! 


Best regards,
Andrej

**Comment by magicus** [comment]
(to be clear, I'm not developing this integration; I'm just an engaged user)

Thanks for the information that 1.14 works! If I have the time, I might have a look at 1x00031 and see if I can submit a PR to add it as a sensor.

**Comment by toringer** [comment]
Tested and verified on firmware version 1.14.
